cli/cli.py

```diff
@@ -0,0 +1,258 @@
+        # Syslog addresses either take the form of a tuple (host, port)
+        # or a string (e.g. "/dev/log"). Here we default to the string
+        # format but use a tuple instead if a port is provided.
+        if syslog_port:
+            syslog_address = (syslog_address, syslog_port)
+        handler = SysLogHandler(address=syslog_address, facility=syslog_facility)
+    elif output:
+        file_path = os.path.abspath(output)
+        handler = logging.FileHandler(file_path, mode="w", encoding="utf-8")
+        handler.setFormatter(logging.Formatter("%(message)s"))
+    else:
+        handler = logging.StreamHandler(sys.stdout)
+        handler.setFormatter(logging.Formatter("%(message)s"))
+    logger.addHandler(handler)
+    logger.setLevel(logging.INFO)
+
+
+def _process_log(log_lines: list, ss_base_url: str, suppress_unspecified: bool):
+    """Process log and return statistics and errors.
+
+    :param log_file: List of nginx access log lines
+    :param ss_base_url: Base URL of Storage Service
+    :param suppress_unspecified: Flag indicating whether to suppress
+        default events
+
+    :returns: Tuple of process statistics (dict) and errors (list)
+    """
+    stats = {EVENTS_WRITTEN: 0, EVENTS_SUPPRESSED: 0, LINES_FILTERED: 0}
+    errors = []
+
+    for line in log_lines:
+        try:
+            parsed_line = parse_access_log_line(line)
+
+            # Empty dict means line was filtered out by URL.
+            if not parsed_line:
+                stats[LINES_FILTERED] += 1
+                continue
+
+            if suppress_unspecified:
+                if parsed_line.get(EVENT_SIGNATURE) == DEFAULT_EVENT[EVENT_SIGNATURE]:
+                    stats[EVENTS_SUPPRESSED] += 1
+                    continue
+
+            cef_event = write_cef_event(parsed_line)
+            logger.info(cef_event)
+            stats[EVENTS_WRITTEN] += 1
+        except (DateParseError, LogParseError) as err:
+            errors.append(err)
+    return (stats, errors)
+
+
+@click.group()
+def auditmatica():
+    """Auditmatica: Archivematica auditing package"""
+    pass
+
+
+@auditmatica.command("write-cef")
+@click.argument("log", type=click.File("r", encoding="utf-8"), default=sys.stdin)
+@click.option(
+    "-o",
+    "--output",
+    type=click.Path(),
+    default=None,
+    help="Filepath for output CEF file (default=None, print to stdout)",
+)
+@click.option(
+    "-s", "--syslog", is_flag=True, help="Write CEF events to syslog instead of file"
+)
+@click.option(
+    "--syslog-address",
+    type=str,
+    default="/dev/log",
+    help="Address for syslog connection (default='/dev/log')",
+)
+@click.option(
+    "--syslog-facility",
+    type=str,
+    default="USER",
+    help="Facility for syslog messages (default='USER')",
+)
+@click.option(
+    "--syslog-port", type=int, default=None, help="Port for remote syslog connections"
+)
+@click.option(
+    "--ss-base-url",
+    type=str,
+    default="http://127.0.0.1:62081",
```

Comment:
```suggestion
    default=DEFAULT_SS_BASE_URL,
```
It'd be cool to share some of these values about. As we get into different configurations or future code changes we'll want just one place to configure these defaults/change them in code.